### PR TITLE
Fix --data-raw body unsuccessful import bug

### PR DESCRIPTION
**Pull request by shreys7**
Fixes the following issues:
- https://github.com/postmanlabs/postman-app-support/issues/8126
- https://github.com/postmanlabs/postman-app-support/issues/7983
- https://github.com/postmanlabs/postman-app-support/issues/7895

Registered the `--data-raw` parameter in commander, add support to populate body.

### CODE REVIEW

File: test/conversion.test.js [hunk 1]
```diff
@@ -356,4 +356,50 @@ describe('Curl converter should', function() {
     expect(header.value).to.equal('$v1');
     done();
   });
+
+  it('[GitHub #8126] [GitHub #7983] [GitHub #7895]: should import body data with --data-raw argument', function (done) {
+
+    // #8126 content-type application/json, so mode = raw
+    var result = Converter.convertCurlToRequest(`curl --location --request PUT
```
Comment (abhijitkane) [inline test/conversion.test.js]: @shreys7 we don't need the exact request the user reported. The test case should be as small as possible.

**Comment by abhijitkane** [comment]
The curl manpage says `(HTTP) This posts data similarly to -d, --data but without the special interpretation of the @ character.`
Are we not handling `--data` correctly, or have we missed something here?

File: test/conversion.test.js (hunk 1 above)
Comment (shreys7) [reply test/conversion.test.js]: Okay. Will change the tests.

**Comment by shreys7** [comment]
> The curl manpage says `(HTTP) This posts data similarly to -d, --data but without the special interpretation of the @ character.`
> Are we not handling `--data` correctly, or have we missed something here?

I did not get you. We are handling it correctly.

**Pull request merged**